code/__DEFINES/color_defines.dm

```diff
@@ -157,3 +157,20 @@
 
 /// Color for dead external organs/zombies
 #define	COLORTONE_DEAD_EXT_ORGAN "#0A3200"
+
+
+/// If this is a plain atom color
+#define ATOM_COLOR_TYPE_NORMAL "normal"
+/// If this is a color filter
+#define ATOM_COLOR_TYPE_FILTER "filter"
+// Indexes for color arrays
+#define ATOM_COLOR_VALUE_INDEX 1
+#define ATOM_COLOR_TYPE_INDEX 2
```

Comment:
Could you make this change to these variables so it's clearer that they don't conflict with the other ones?
```suggestion
#define ATOM_COLOR_INDEX_VALUE 1
#define ATOM_COLOR_INDEX_TYPE 2
```